flutter/news/lib/features/frontpage/data/datasource/articles_remote_data_source.dart

```diff
@@ -9,10 +9,10 @@ class ArticlesRemoteDataSource {
 
   Future<Result<List<Article>>> topHeadLines() async {
     var result = await client
-        .topHeadLines()
+        .topHeadLines("us")
```

Comment:
I think this could read very well with a named argument for the `topHeadlines` method. `topHeadlines(country: "us")